README.md

```diff
@@ -1,25 +1,108 @@
 # Quarkus Backstage
 
-[![Version](https://img.shields.io/maven-central/v/io.quarkiverse.backstage/quarkus-backstage?logo=apache-maven&style=flat-square)](https://search.maven.org/artifact/io.quarkiverse.backstage/quarkus-backstage)
+[![Version](https://img.shields.io/maven-central/v/io.quarkiverse.backstage/quarkus-backstage?logo=apache-maven&style=flat-square)](https://central.sonatype.com/artifact/io.quarkiverse.backstage/quarkus-backstage-parent)
 
-## Welcome to Quarkiverse!
+Generate Backstage Catalog Information as part of the Quarkus build or the Quarkus CLI.
 
-Congratulations and thank you for creating a new Quarkus extension project in Quarkiverse!
+## Features
 
-Feel free to replace this content with the proper description of your new project and necessary instructions how to use and contribute to it.
+- Generate the catalog-info.yaml for the Quarkus application
+- Command Line interface to install / uninstall and Component to Backsatge
+- Orchestrate Quarkus extensions:
+  - kubernetes
+  - helm
+  - argocd
 
-You can find the basic info, Quarkiverse policies and conventions in [the Quarkiverse wiki](https://github.com/quarkiverse/quarkiverse/wiki).
+## Requirements
 
-In case you are creating a Quarkus extension project for the first time, please follow [Building My First Extension](https://quarkus.io/guides/building-my-first-extension) guide.
+### For Catalog Info Generation
+To generate the catalog-info.yaml nothing special is required.
 
-Other useful articles related to Quarkus extension development can be found under the [Writing Extensions](https://quarkus.io/guides/#writing-extensions) guide category on the [Quarkus.io](https://quarkus.io) website.
+### For using the CLI / Client
+- A running Bacsktage installation with a known token
```

Comment:
A running Bacsktage installation with a known token =>
A running Backstage instance with a known token

Remark: Are you talking about this token https://backstage.io/docs/auth/service-to-service-auth#static-tokens ? If yes, please add the link to the backstage documentation